src/main/java/pers/gwyog/gtneioreplugin/util/GT5OreLayerHelper.java

```diff
@@ -26,6 +45,30 @@ public GT5OreLayerHelper() {
         for (OreLayerWrapper layer : mapOreLayerWrapper.values()) {
             bufferedDims.put(layer, getDims(layer));
         }
+
+        // ------------------------------
+        // Get dims as "Ow,Ne,Ma" etc.
+        for (OreLayerWrapper oreLayer : bufferedDims.keySet()) {
+
+            String dims = bufferedDims.get(oreLayer);
+
+            if (dims.equals("Not available in any Galactic Dim!")) {
```

Comment:
I guess this exact String can be found at least at one other place, so it's a good idea to define a constant here or there, so shit doesn't break if one or the other is changed.